src/modality-specific-files/near-infrared-spectroscopy.md

```diff
@@ -22,7 +22,7 @@ and a guide for using macros can be found at
 {{ MACROS___make_filename_template(
    "raw",
    datatypes=["nirs"],
-   suffixes=["nirs", "events", "physio", "stim"])
+   suffixes=["nirs", "events", "channels", "optodes", "coordsystem", "physio", "stim", "eyetrack"])
```

Comment:
This should be fixed directly in the spec's repo.
```suggestion
   suffixes=["nirs", "events", "physio", "stim", "eyetrack"])
```

Reply:
yup indeed